src/Navbar.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './Navbar.css';
+//import logo from 'logo-space'
+//import search image from 'search-image'
+
+const Navbar = (props) => {
+  return (
+    <div className='navbar'>
+      <h1>Tomates Podres 'logo'</h1>
+      {props.backToHome && <button onClick={() => props.returnHome()}>Return to Home</button>}
+      {!props.backToHome && <p>Searchbar logo</p>}
+    </div>
+  )
+}
+
+export default Navbar;
```

Comment:
Felling very proud of our code! 
I Really like our pseudocoding and how quick we have been able to put our thoughts together !